app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from app.errors import NotVaccinatedError
```

Comment:
No need to have import lines. Combine into one

Reply:
it is too long (more than 79) if I create one line That why i import classes in 3 lines
Or is it a mistake?